manual/src/welcome.md

```diff
@@ -0,0 +1,26 @@
+# Welcome
+
+**OrbTk is an UI-Toolkit to write scalable cross platform graphical user interface in the programming language Rust.**
+
+We are glad you are here and interested in OrbTk. The aim of this manual is to help you with your first steps with OrbTk. 
+It will guide you step by step how to write you first applications with OrbTk and will help you to understand the core concepts
+of OrbTk.
+
+*Hint:* This manual is work in progress and more content will produced step by step.
```

Comment:
"manual is work" --> "manual is a work"; "content will produced" --> "content will be produced"